frontend/src/components/question-creation/HoverMessage.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { Text } from "@chakra-ui/react";
+
+const HoverMessage = (): React.ReactElement => <Text>You are hovering.</Text>;
+
+export default HoverMessage;
```

Comment:
This is temporary, I'm waiting for a design decision on what the Hover state should look like here!